### DISCUSSION TITLE
Autoplay avatar

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#14987] [Ideas] [answered]
Change current autoplay avatar to autoplay mod icon or osu! logo.

![ScreenShot_20211006203547](https://user-images.githubusercontent.com/86934170/136256045-9cfa4ad6-fef8-44ff-956b-baa38aaa3c6b.png)

**Comment by Joehuu** [comment, marked as answer]
https://github.com/ppy/osu/issues/15237
